src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultIconInput/Components/defaultPasswordInput/index.tsx

```diff
@@ -36,7 +35,7 @@ export const DefaultPasswordInput = ({
         <I.Input
           type={type}
           placeholder={placeholder}
-          value={state}
```

Comment:
I place it there intentionally see https://deploy-preview-813--peaceful-nightingale-55a1b7.netlify.app/signup/student try to signup, once you submit values on input fields won't erase. because now input value is equal to value, not to state. try & experiment you will understand what I meant.

Reply:
Got it. Thanks for pointing out.